extensions/amp-animation/0.1/amp-animation.js

```diff
@@ -154,8 +154,11 @@ export class AmpAnimation extends AMP.BaseElement {
     }
 
     // Actions.
-    this.registerAction('start',
-        this.startAction_.bind(this), ActionTrust.LOW);
+    this.registerAction(
+        'start',
+        this.startAction.bind(this),
+        ActionTrust.LOW,
+        /* opt_isDefault */ true);
```

Comment:
`registerDefaultAction`?

To avoid allowing this, maybe remove the optional param and have both `registerAction` and `registerDefaultAction` call an internal method.